kubeyaml.py

```diff
@@ -39,19 +39,31 @@ def note(s):
 def yaml():
     y = YAML()
     y.explicit_start = True
+    y.explicit_end = False
     y.preserve_quotes = True
     return y
 
 def bail(reason):
         sys.stderr.write(reason); sys.stderr.write('\n')
         sys.exit(2)
 
+class AlwaysFalse(object):
+    def __init__(self):
+        pass
+
+    def __get__(self, instance, owner):
+        return False
+
+    def __set__(self, instance, value):
+        pass
+
 def apply_to_yaml(fn, infile, outfile):
     # fn :: iterator a -> iterator b
     y = yaml()
+    # Hack to make sure no end-of-document ("...") is ever added
+    y.Emitter.open_ended = AlwaysFalse()
```

Comment:
Why do we need this _and_ `y.explicit_end = False`?

Reply:
To be fair, I didn't really understand the implications of `y.explicit_end = False`, I think that we can get the same result without `y.explicit_end = False`.

However, if I recall properly, without `y.Emitter.open_ended = AlwaysFalse()` it will still output an ending `...`.

Reply:
Belt and braces ..